JapaneseFilter/sections/specific.txt

```diff
@@ -39,6 +39,7 @@ financial-field.com,spaia.jp,yomiuri.co.jp,fujinkoron.jp,mikle.jp,onayamifree.co
 !
 !
 cosme.net##.atcosme-HeaderAdContent
+baby-calendar.jp###sp
```

Comment:
Where this is used?

Reply:
Unfortunately, I can't find `baby-calendar.jp##div[id^="JP_BABY_sp_all_"]` and `baby-calendar.jp##.linkArea` on this page. Can I ask you to provide a screenshot?

Reply:
Found false positive by `##.linkArea` so no need.
<details>
<summary>Screenshots</summary>

![baby1](https://user-images.githubusercontent.com/58900598/160611815-f39eacbc-e54b-4414-be42-233f6fd7a83d.png)

![baby2](https://user-images.githubusercontent.com/58900598/160611858-e385e01e-f28f-48c1-97bf-d71c4bb019c1.png)

</details>

Reply:
Directly added `baby-calendar.jp##div[id^="JP_BABY_sp_all_"]`